content/en/docs/tasks/administer-cluster/change-pv-access-mode-readwriteoncepod.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Change the Access Mode of a PersistentVolume to ReadWriteOncePod
+content_type: task
+weight: 90
+---
+
+<!-- overview -->
+This page shows how to change the access mode on an existing PersistentVolume to
+use ReadWriteOncePod.
+
+## {{% heading "prerequisites" %}}
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+{{< note >}}
+The ReadWriteOncePod access mode is only supported for CSI volumes and
+Kubernetes version 1.22+. To use this this feature you will need to update the
+following [CSI sidecars](https://kubernetes-csi.github.io/docs/sidecar-containers.html)
+to these versions or greater:
```

Comment:
```suggestion
The ReadWriteOncePod access mode is only supported for
{{< glossary_tooltip text="CSI" term_id="csi" >}} volumes.
To use this volume access mode you will need to update the
```
(use `min-kubernetes-server-version` in front matter to specify the required version).